src/categorical_algebra/DataMigrations.jl

```diff
@@ -268,6 +273,9 @@ SigmaMigration(f,d::Type{T′},::Type{T}) where {T<:StructACSet, T′<:StructACS
 """
 Create a C-Set for the collage of the functor. Initialize data in the domain 
 portion of the collage, then run the chase.
+
+The true result data is a diagram morphism. We return the result ACSet and the 
+diagram map data as a dictionary with FinFunctions for each Ob.
```

Comment:
I believe this is the unit of the adjunction between the sigma migration and delta migration functors. In particular, it is the data of an acset transformation on the domain schema C of the functor F: C -> D. We should document that.

Ideally we would return this as a diagram morphism from C-Set to D-Set whose functor part is F, rather than as a dict. Would that be impractical?